docs/notebooks/Distributed_arrays_and_automatic_parallelization.md

```diff
@@ -13,19 +13,28 @@ kernelspec:
 
 +++ {"id": "PxHrg4Cjuapm"}
 
-# Parallelism with JAX
+# Distributed arrays and automatic parallelization
 
 +++ {"id": "pFtQjv4SzHRj"}
 
 **This tutorial discusses parallelism via `jax.Array`, the unified array object model available in JAX v0.4.0 and newer.**
 
-See {ref}`jax-array-migration` guide for migrating existing pre-v0.4.0 codebases to `jax.Array`.
```

Comment:
Let's keep the ref?

Reply:
Hrm, I made it a markdown link so it would work in the ipynb (as {ref} wasnt working in Colab). Is there some way to make {ref} work in ipynb/colab too?

Reply:
Jake told me that it doesn't work in colab bit for website it works and it works in staging rtd though.

I am fine either way though